test/extended/authorization/per_node_validatingadmissionpolicybinding.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: ValidatingAdmissionPolicyBinding
+metadata:
+  name: "only-allow-name-matching-node-configmaps"
+spec:
+  policyName: "only-allow-name-matching-node-configmaps"
+  validationActions: [Deny]
+  matchResources:
+    namespaceSelector:
+      matchLabels:
+        kubernetes.io/metadata.name: "e2e-ns"
```

Comment:
this is currently bound to a single namespace, but you could bind to anything or all.  Just kept it small for the test.